docs/index.html

```diff
@@ -690,7 +694,11 @@ <h1 class="display-4 theme-content-element-title">Cards</h1>
 		<span class="theme-content-element-source-label">Source:</span>
 		<span class="theme-content-element-source-path">cards.md</span>
 	</span>
-	<div class="theme-content-element-description"><p>In component based services (especially internal systems), content can be structured using Cards.</p>
+	<div class="theme-content-element-description"><p>A card is composed of several elements (eg title, description, icon, illustration). Combined, these elements are related to a single theme or to a destination. A card does not have an active / selected state. Cards contain interactive content, like function or other action points. Since cards are the starting point for more detailed info, they should contain a limited number of actions.</p>
+<h3>✅ When to use:</h3><p>Cards are used when you want to view content related to a theme. The text of the card is recommended to be kept at a maximum of 70 characters per line to improve the readability for the user. The content should be short and kept to a minimum. Padding inside the card is recommended to have spacing-3. If there are multiple cards in a row, the cards should have the same height. </p>
+<p>The card can have an icon in the label to improve the recognition of content in the top left corner. This should be inside a circle, aligned to the H1 element The icon should clearly be connected to the title, and be relevant to content </p>
+<p>A card can have an icon prompting the user to go to settings for this specific content. If a card is clickable, this should be displayed with a light border, view more and an arrow.</p>
+<h3>⛔ When not to use:</h3><p>Cards should not be used when the user does not directly compare images or text.</p>
```

Comment:
Så kort skal KUN brukes for sammenligning?

Reply:
Det är en rekommendation från Google Material Design relaterat till scannability

_"The use of cards isn’t recommended when “a user is not directly comparing images or text.” Users of a news website often want to compare news titles especially on the homepage, to get an overview of the current stories and decide which articles they want to explore further."_

Då det handlar om att jämföra olika ting som tillhör samma grupp för eksempel. Mer läsning:
https://medium.theuxblog.com/when-card-ui-design-doesnt-work-f4343118d108

Reply:
Virker som google snakker om kort som en erstatning for en tabell eller liste. Vi bruker kort til mye annet, og gjerne standalone. Enten må vi definere utifra vår bruk, eller endre vår bruk av kort. Virker mer som kort skal bare være en into til noe mer, men vi bruker det til hele løsningen

https://uxplanet.org/best-practices-for-cards-fa45e3ad94dd

Reply:
Yeah, ser den. Ta bort den kommentaren då :)

Reply:
Er det noen setting kort ikke skal brukes isåfall?

Reply:
Er noe vi må finne ut av 🤔